etna/analysis/plotters.py

```diff
@@ -177,7 +177,7 @@ def plot_forecast(
 
         # plot forecast plot for each of given forecasts
         quantile_prefix = "target_"
-        for j, (forecast_name, forecast) in enumerate(forecast_results.items()):
+        for _j, (forecast_name, forecast) in enumerate(forecast_results.items()):
```

Comment:
Why?

Reply:
`B007: Loop control variable not used within the loop body. If this is intended, start the name with an underscore.`
https://github.com/PyCQA/flake8-bugbear

Reply:
Should we use in that case:
```python
for forecast_name, forecast in forecast_results.items():
```

Reply:
I agree that it will have more sense and change it in the next commit.

Reply:
Fixed